build.gradle

```diff
@@ -2,8 +2,23 @@
 
 buildscript {
     repositories {
-        google()
-        jcenter()
+        if (project.hasProperty("googleRepo")) {
+            maven {
+                name "Google"
```

Comment:
Setting names allows to get a consistent output. For instance, if no repo is overridden, this is what you get:

```
> Task :listRepositories
Repositories:
Name: Google; url: https://dl.google.com/dl/android/maven2/
Name: Mozilla Nightly; url: https://nightly.maven.mozilla.org/maven2
Name: Mozilla; url: https://maven.mozilla.org/maven2
Name: BintrayJCenter; url: https://jcenter.bintray.com/
```

Without a name set, you end up having:


```
> Task :listRepositories
Repositories:
Name: Maven; url: http://localhost:8081/google
Name: Maven 2; url: http://localhost:8081/nightly
Name: Maven 3; url: http://localhost:8081/maven-mo
Name: Maven 4; url: http://localhost:8081/jcenter
```